packages/block-editor/src/components/default-block-appender/style.scss

```diff
@@ -12,34 +12,9 @@
 		outline: 1px solid transparent;
 	}
 
-	textarea.block-editor-default-block-appender__content { // Needs specificity in order to override input field styles from WP-admin styles.
-		font-family: inherit;
-		font-size: inherit;
-		border: none;
-		background: none;
-		box-shadow: none;
-		display: block;
-		cursor: text;
-		width: 100%;
-		outline: $border-width solid transparent;
-		transition: 0.2s outline;
-		@include reduce-motion("transition");
-		margin-top: $default-block-margin;
-		margin-bottom: $default-block-margin;
-
-		// This needs high specificity as it's output as an inline style by the library.
-		resize: none !important;
-
-		// Emulate the dimensions of a paragraph block.
-		// On mobile and in nested contexts, the plus to add blocks shows up on the right.
-		// The rightmost padding makes sure it doesn't overlap text.
-		padding: 0 #{ $block-padding + $button-size } 0 0;
-
-		// Use opacity to work in various editor styles.
-		color: $dark-gray-placeholder;
-		.is-dark-theme & {
-			color: $light-gray-placeholder;
-		}
+	.block-editor-default-block-appender__content {
+		// Set the opacity of the initial block appender to the same as placeholder text in an empty Paragraph block.
+		opacity: 0.62;
```

Comment:
Hm, I checked and I think it's fine for now to not create a variable. The value is only used in one other place and there's a comment on the reason for this value. Maybe we should use a class instead of a variable? Cc @jasmussen 

In any case, we can iterate.